lib/teaching_vacancies_api.rb

```diff
@@ -0,0 +1,19 @@
+module TeachingVacancies
+  class API
+    BASE_URL = 'https://teaching-vacancies.service.gov.uk/api/v1/'.freeze
+
+    def jobs(limit: 50)
+      response = HTTParty.get(api_url('jobs.json'))
+      json = JSON.parse(response.body)
+      json['data'].take(limit)
+    rescue HTTParty::ResponseError => e
+      Rails.logger.warn("Teaching Vacancies API response error: #{e.message}")
```

Comment:
We probably want to raise the error and handle it in the Rake task too, as it won't return an array and it won't immediately be clear what's happened

Reply:
I've added a commit that moves the `rescue` to the rake task, thanks for pointing it out 👍